Content.Tests/DMProject/Tests/Operators/scope_parent_type_fails.dm

```diff
@@ -0,0 +1,10 @@
+// COMPILE ERROR OD0011
+
+/datum/thing
+	var/price = 60
+	better
+		proc/test_proc()
+			price = parent_type::price + 40
+
+/proc/RunTest()
+	var/datum/thing/better/test
```

Comment:
```suggestion
```
doesn't do anything

Reply:
Good point - apparently I can also get rid of `/proc/RunTest()` entirely and it works as expected

confirmed by changing to `// COMPILE ERROR OD0012` and the test case fails.